internal/dag/httpproxy_processor.go

```diff
@@ -405,6 +406,10 @@ func (p *HTTPProxyProcessor) computeRoutes(
 		delete(p.orphaned, types.NamespacedName{Name: includedProxy.Name, Namespace: includedProxy.Namespace})
 	}
 
+	dynamicHeaders := map[string]string{
+		"CONTOUR_NAMESPACE": proxy.Namespace,
```

Comment:
Seems like this is the only dynamic header added so far? I can't see any sign of the other two, `%CONTOUR_SERVICE_NAME%`, and `%CONTOUR_SERVICE_PORTR%`?

Reply:
They're set on lines 567 and 568 below where the service name and port are available.

Reply:
Ah, so they are, thanks.